org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/hover/FocusableBrowserInformationControl.java

```diff
@@ -91,21 +91,22 @@ protected void createContent(Composite parent) {
 			@Nullable
 			Point constraints = getSizeConstraints();
 			Point hint = computeSizeHint();
-
 			setSize(hint.x, hint.y);
-			browser.execute("document.getElementsByTagName(\"html\")[0].style.whiteSpace = \"nowrap\""); //$NON-NLS-1$
-			Double width = 20 + (Double) browser.evaluate("return document.body.scrollWidth;"); //$NON-NLS-1$
 
+			safeExecute(browser, "document.getElementsByTagName(\"html\")[0].style.whiteSpace = \"nowrap\""); //$NON-NLS-1$
+			Double width = (Double) safeEvaluate(browser, "return document.body.scrollWidth;"); //$NON-NLS-1$
+			width = 20 + (width != null ? width : 0);
```

Comment:
Casting here is somewhat dangerouns, I'd suggest
```suggestion
			double width = 20 + (safeEvaluate(browser, "return document.body.scrollWidth;") instanceof Double evaluated ? evaluated : 0); //$NON-NLS-1$
```
which should prevent some classCast from happening if browser returns some other number type one day.